### PR TITLE
test get object with special key name

### PR DESCRIPTION
**Pull request by oritwas**
Signed-off-by: Orit Wasserman <owasserm@redhat.com>

### CODE REVIEW

File: s3tests/functional/test_s3.py [hunk 1]
```diff
@@ -4707,6 +4707,11 @@ def test_bucket_create_special_key_names():
     names = [e.name for e in list(li)]
     eq(names, key_names)
 
+    for name in key_names:
+        key = bucket.get_key(name)
```
Comment (andrewgaul) [inline s3tests/functional/test_s3.py]: Check that `key.name` is equal to `name`.

**Comment by oritwas** [comment]
@andrewgaul , added

**Comment by andrewgaul** [comment]
:+1:

**Comment by oritwas** [comment]
@yehudasa , can you merge this pr?

**Pull request merged**